WTWSMS/history/titles/d_auvergne.txt

```diff
@@ -1,3 +1,10 @@
 466.1.1={ 
 	holder=30 #Visigoths
+}
```

Comment:
30 dies in 507, I think the title can be without holder between 507-525.

Reply:
30 dies in 484, but solution remains the same.